<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QQLogin</class>
 <widget class="QDialog" name="QQLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>281</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>macqq.png</normaloff>macqq.png</iconset>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
background-image: url(:/new/login/images/LoginPanel/Background/login_bg.png);
border:1px solid white;
border-radius:5px;
padding:2px 4px;
}
</string>
  </property>
  <widget class="QPushButton" name="pb_login_">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:1px solid gray;
border-radius:5px;
}

QPushButton:focus{
border:1px solid gray;
border-radius:5px;
background-image: url(:/new/login/images/LoginPanel/btn_focus.png);
}

QPushButton:hover{
border:1px solid gray;
border-radius:5px;
background-image: url(:/new/login/images/LoginPanel/btn_highlight.png);
}

QPushButton:pressed{
border:1px solid gray;
border-radius:5px;
background-image: url(:/new/login/images/LoginPanel/btn_down.png);
}</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/login/images/LoginPanel/bg_white.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/new/login/images/LoginPanel/Main_Title.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>181</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/login/images/LoginPanel/bg_white.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="img.qrc">:/new/login/images/LoginPanel/StdLoginBanner.png</pixmap>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_mini_">
   <property name="geometry">
    <rect>
     <x>307</x>
     <y>-1</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
border-image: url(:/new/login/images/LoginPanel/btn_mini_normal.png);
}

QToolButton:hover{
border-image: url(:/new/login/images/LoginPanel/btn_mini_highlight.png);
}

QToolButton:pressed{
border-image: url(:/new/login/images/LoginPanel/btn_mini_down.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_close_">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>-1</y>
     <width>35</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{
border-top-right-radius:5px;
background-image: url(:/new/login/images/LoginPanel/btn_close_normal.png);
}

QToolButton:hover{
border-top-right-radius:5px;
background-image: url(:/new/login/images/LoginPanel/btn_close_highlight.png);
}

QToolButton:pressed{
border-top-right-radius:5px;
background-image: url(:/new/login/images/LoginPanel/btn_close_down.png);
}
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="popupMode">
    <enum>QToolButton::DelayedPopup</enum>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_password_">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>180</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/login/images/LoginPanel/bg_white.png);
border:1px solid gray;
border-radius:2px;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>online</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>leave</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>busy</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>hide</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>don't disturb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>free</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="le_username_">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>191</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/login/images/LoginPanel/bg_white.png);
border:1px solid gray;
border-radius:2px;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>84</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/login/images/LoginPanel/bg_white.png);</string>
   </property>
   <property name="text">
    <string>Auto Login</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/login/images/LoginPanel/bg_white.png);</string>
   </property>
   <property name="text">
    <string>Remember Password</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_username_</tabstop>
  <tabstop>le_password_</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>checkBox</tabstop>
  <tabstop>checkBox_2</tabstop>
  <tabstop>pb_login_</tabstop>
  <tabstop>tb_close_</tabstop>
  <tabstop>tb_mini_</tabstop>
 </tabstops>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
